project/src/main/java/com/google/sps/Servlets/CommentServlet.java

```diff
@@ -52,3 +57,15 @@ private String convertToJson(Object obj) {
     return json;
   }
 }
+
+class CommentWithName {
```

Comment:
why can't you just modify the comment class to have a name field that has an associated getter / setter?

Reply:
then in your for loop when you're associating the name you don't need a new class

Reply:
That would also work but I would have to go back and refactor several functions in the database, I think our time is better spend fixing other bugs and nitpicks right now